pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "grass_gis_helpers"
+version = "0.0.1"
+authors = [
+  { name="Anika Weinmann", email="weinmann@mundialis.de" },
+]
+maintainers = [{name="mundialis GmbH & Co. KG", email="info@mundialis.de"}]
+description = "This library includes functions which mundialisuses in different GRASS GIS addons."
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: OS Independent",
+]
+dependencies = []
```

Comment:
```suggestion
dependencies = ["grass"]
```

I guess the `grass` module needs to be added to the requirements:

```
09:21 $ docker run --rm -it --entrypoint sh alpine
/ # apk add python3 py3-pip
[...]
/ # pip3 install grass-gis-helpers
Collecting grass-gis-helpers
  Downloading grass_gis_helpers-0.0.0-py3-none-any.whl (19 kB)
Installing collected packages: grass-gis-helpers
Successfully installed grass-gis-helpers-0.0.0
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
/ # python3
Python 3.10.12 (main, Jun  9 2023, 02:29:43) [GCC 12.2.1 20220924] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from grass_gis_helpers import cleanup
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/site-packages/grass_gis_helpers/cleanup.py", line 21, in <module>
    import grass.script as grass
ModuleNotFoundError: No module named 'grass'
```

For `grass` itself - or a requirement of it - packages like `make, gcc, python3-dev, ...` are needed. We might add this information to the README?

Reply:
Ah - one needs to be in a GRASS GIS session anyway. So it is not required. Maybe this information should be in the README then.